lib/c-api/src/ast_yaml.cc

```diff
@@ -1,5 +1,10 @@
 #include <clingo/ast.h>
 
+#ifdef _WIN32
```

Comment:
You can get the yaml by splitting the string literal into three.
I.e. something like:
```cpp
R"yaml(unary_operator:
...
)yaml"
R"yaml(term_unary_operation:
...
)yaml"
...
```